integrated_channels/moodle/client.py

```diff
@@ -474,6 +485,10 @@ def create_course_completion(self, user_id, payload):
         # The base integrated channels transmitter expects a tuple of (code, body),
         # but we need to wrap the requests
         resp = self._wrapped_create_course_completion(user_id, payload)
+        LOGGER.info(
+            'Response for Moodle Create Course Completion Request '
+            f' response: {resp} '
+        )
```

Comment:
can we use `generate_formatted_log` utility here?

Reply:
@hamzawaleed01 updated.